packages/events-example/src/deployments/pulumi.ts

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
it would be great if the class and the filename were the same

Reply:
Stylistically, do we prefer a class per file? I've got that interface in here as well. I don't have a strong opinion either way (class-per-file vs. multi-class files).

Reply:
I personally do not either, but I think for the purposes here of exploration, it makes it easier

Reply:
my thought was since we're doing 
```
import PulumiAwsDeployment from '../../deployments/pulumi';
```
this should follow that, since it is the default export.